sky/cli.py

```diff
@@ -3031,7 +3034,11 @@ def spot_cancel(name: Optional[str], job_ids: Tuple[int], all: bool, yes: bool):
 @usage_lib.entrypoint
 def spot_logs(name: Optional[str], job_id: Optional[int], follow: bool):
     """Tail the log of a managed spot job."""
-    core.spot_tail_logs(name=name, job_id=job_id, follow=follow)
+    try:
+        core.spot_tail_logs(name=name, job_id=job_id, follow=follow)
+    except exceptions.ClusterNotUpError:
+        # Hint messages already printed by the call above.
+        pass
```

Comment:
Can we have the hint message as the message of the exception instead, or do we want to exit with a non-zero return code?

Reply:
Added `sys.exit(1)`. For CLI commands, we probably don't want to print the exceptions, wdyt?